base/.eslintrc.json

```diff
@@ -49,10 +48,11 @@
 
     "radix": 2,
     "curly": 2,
-    "no-fallthrough": 0,
-    "default-case": 0,
-    "no-plusplus": 0,
-    "dot-notation": 0,
-    "eol-last": 2
+    "eol-last": 2,
+    "no-else-return": 2,
```

Comment:
It's actually to avoid unnecessary conditions essentially... you should return at the earliest point possible, rather than hitting all conditions first. I'll try to think of an example.